### PR TITLE
Add  scikit-rf

### PR DESCRIPTION
**Pull request by arsenovic**
first attempt to add the package  scikit-rf to conda-forge

### CODE REVIEW

File: recipes/scikit-rf/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+
```
Comment (ocefpaf) [inline recipes/scikit-rf/meta.yaml]: Please remove this extra line.

File: recipes/scikit-rf/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+{% set name = "scikit-rf" %}
+{% set version = "0.14.3" %}
+{% set compress_type = "zip" %}
+{% set hash_type = "md5" %}
+{% set hash_val = "3dc7e7aab5e0dd1f5ce692f32b053ec7" %}
+{% set build_num = "0" %}
```
Comment (ocefpaf) [inline recipes/scikit-rf/meta.yaml]: I really do not recommend all that `jinja`. Only the version is enough. (Ultimately that is your decision.)

File: recipes/scikit-rf/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: {{ build_num }}
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - pandas
```
Comment (ocefpaf) [inline recipes/scikit-rf/meta.yaml]: Most of the time, when using `--single-version-externally-managed`, you don't need to specify `run` time dependencies at `build` time. Try to remove them and see if it works. (Sometimes packages import themselves in the `setup.py` and you do need them though.)

File: recipes/scikit-rf/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    - setuptools
+    - ipython
+    - numpy
```
Comment (ocefpaf) [inline recipes/scikit-rf/meta.yaml]: Does this relies on `numpy`'s ABI? If so you need `numpy x.x` at both `build` and `run` to bake the `numpy` build string in the package.

File: recipes/scikit-rf/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+test:
+  imports:
+    - skrf
+    - skrf.calibration
+    - skrf.data
+    - skrf.io
+    - skrf.media
+
+
```
Comment (ocefpaf) [inline recipes/scikit-rf/meta.yaml]: Extra line.

File: recipes/scikit-rf/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+about:
+  home: http://www.scikit-rf.org
+  license: new BSD
```
Comment (ocefpaf) [inline recipes/scikit-rf/meta.yaml]: New BSD is the `BSD 2-Clause` or `BSD 3-clause`? (We rather use the clause syntax to keep it easier to identify btw.)

File: recipes/scikit-rf/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+  build:
+    - python
+    - pandas
+    - setuptools
+    - ipython
+    - numpy
+    - scipy
+    - matplotlib
+    - mock
+    - nose
```
Comment (ocefpaf) [inline recipes/scikit-rf/meta.yaml]: Do you really need `mock` and `nose` at `build` time? Those are usually `test` dependencies.

File: recipes/scikit-rf/meta.yaml (hunk 4 above)
Comment (arsenovic) [inline recipes/scikit-rf/meta.yaml]: im not sure what numpy ABI is.

File: recipes/scikit-rf/meta.yaml (hunk 1 above)
Comment (arsenovic) [inline recipes/scikit-rf/meta.yaml]: k

**Comment by arsenovic** [comment]
@ocefpaf  , so i gave this another try today. unfortunately i do no not understand how to troubleshoot this error.   i have tried the same build command that i found in the docker script, 

`conda-build-all  scikit-rf-14.4 --matrix-conditions "numpy >=1.11" "python >=2.7,<3|>=3.5" "r-base >=3.3.2"  
`
and  this command works on my machine, but the builds still fail with same error message about  conflict.  any suggestions are much appreciated?

**Comment by ocefpaf** [comment]
Testing locally. Hold on...

**Pull request closed without merging**